phase1/openstack/openstack.jsonnet

```diff
@@ -69,6 +69,17 @@ function(cfg)
             k8s_master_ip: "${openstack_compute_instance_v2.%(master_instance)s.network.0.fixed_ip_v4}" % names,
           },
         },
+       	//Populates openstack config file
+        cloudprovider: {
+          template: "${file(\"openstack.conf\")}",
```

Comment:
@ss22ever, I am little curious where is this template file rendered? How is the `openstack.conf` file created in target VM's filesystem and how the cloud-provider is initialised with this config file.